lib/traject/vatican_iiif_config.rb

```diff
@@ -55,6 +55,13 @@
     "name[@role!='internal' and @role!='external' or not(@role)]/alias/authorityAuthor[@rif='aut']", nil,
     gsub: [/[,\.]$/, '']
   )
+  to_field 'other_name_and_role_ssim', (accumulate do |resource, *_|
+    resource.xpath("name[@role!='internal' and @role!='external' or not(@role)]").flat_map do |name|
+      name.xpath("alias/authorityAuthor[@rif='aut']").map do |author|
+        "#{author.text.gsub(/[,\.]$/, '')} [#{name['role']}]"
```

Comment:
Do we know that all names here will have a role? (or maybe I'm confused about what `not(@role)` above would include in xpath).

Reply:
It seemed like all (many?) of these other names had roles. I have no clue about the xpath.

Reply:
Nope, I've found some counter examples.. I guess I'll make this formatting conditional.